javascript-data-model/order-history.js

```diff
@@ -0,0 +1,61 @@
+const orderHistory = [
+  {
+    'Aug 8, 2020': [
+      {
+        'order #': 11439416898662232,
+        category: 'book',
+        title: 'Javascript for impatient programmers',
+        author: 'Rauschmayer, Dr Axel',
+        price: 31.55,
+        'return window': 'Sep 7 2020',
```

Comment:
As we touched on in our Q&A, our property names should be using `camelCased`. So this should become `returnWindow` instead